.github/workflows/release-cycle.yml

```diff
@@ -19,10 +19,17 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: "3"
+          cache: pip
+
+      - name: Install Jinja
+        run: python -m pip install jinja2
 
       - name: Generate release cycle output
         run: python -I -bb -X dev -X warn_default_encoding -W error _tools/generate_release_cycle.py
 
+      - name: Ignore release-cycle.svg, which encodes the current date
+        run: git restore include/release-cycle.svg
+
```

Comment:
Maybe just add the ability to pass a fixed date to the script to be able to get reproducible output and still be able to check this?

Reply:
I've added an override to be able to pass a fixed date.

But of course this still fails because we're generating for a fixed date, but the one in Git was last generated on some other, arbitrary date...

https://github.com/python/devguide/actions/runs/3979549956/jobs/6822048311

Reply:
Oh, right—duh, silly me! Maybe store the date as a string in a comment in the first or second line of the SVG and read it out of there?

Reply:
Well, and now the SVG is generated for every deploy and removed from Git (https://github.com/python/devguide/pull/1034/commits/59c2bbcb716ab22389951f8bbf875bfa89c97f82), I think we're probably fine with just running the script and making sure it doesn't fail? The other workflow makes sure the file can be generated and included in the docs with no error.

If so, do we need the `--today` override?

Reply:
Just to follow up, we concluded below that since the script is being run (ensuring it doesn't fail) now during the regular CI checks anyway, we can just remove the workflow as its redundant, which you went ahead and did.

And we also discussed just leaving it in other other potential use cases, so I think this can be marked resolved now, thanks!